nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenHTTP.java

```diff
@@ -74,7 +78,10 @@
 @Tags({"ingest", "http", "https", "rest", "listen"})
 @CapabilityDescription("Starts an HTTP Server and listens on a given base path to transform incoming requests into FlowFiles. "
         + "The default URI of the Service will be http://{hostname}:{port}/contentListener. Only HEAD and POST requests are "
-        + "supported. GET, PUT, and DELETE will result in an error and the HTTP response status code 405.")
+        + "supported. GET, PUT, and DELETE will result in an error and the HTTP response status code 405. "
+        + "GET is supported on <service_URI>/healthcheck. If the service is available, it returns \"200 OK\" with an empty response body. "
+        + "The health check functionality can be configured to be accessible via a different port. "
```

Comment:
The "empty response body" is not correct any more.

Reply:
Thank you! Fixed it with the next commit.